avm/src/main.rs

```diff
@@ -131,7 +134,10 @@ fn main() {
           agc_file
         );
         telemetry::log("avm-run").await;
-        run_file(&fp, false).await;
+        if let Err(ee) = run_file(&fp, false).await {
+          eprintln!("{}", ee);
+          std::process::exit(2);
```

Comment:
Why exit code `2` and not `1`?

Reply:
that was what we were using in the majority of cases where we did `eprintln!()` and then `exit()`